File: Plov/src/Stove.java

public class Stove {
	private Pan pan;
	
	private boolean state;
    public boolean GetState(){
    	return state;
    }
    public void SetState(boolean s){
    	state=s;
    }
	
	  public Pan GetPan(){
	    return pan;
	    }
	    public void SetPan(Pan p){
	    	pan=p;
	    }
	    

	
	 public void Cooking()
	    {
	            	
		 pan.Cook();
	        }
	 

}

File: Laba1Plov/Laba1Plov/Stove.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Laba1Plov
{
    class Stove
    {

        private bool state_stove;
        private Pan pan;


        public bool State
        {
            set  { state_stove = value; }
            get { return state_stove; }
        }

        public Pan Pan { set { pan = value; } get { return pan; } }

        public void Cooking()
        {
              pan.Cook();
        }
     

    }
}
